src/elements/ngm-auth.js

```diff
@@ -25,12 +25,32 @@ class NgmAuth extends I18nMixin(LitElement) {
   constructor() {
     super();
     this.user = Auth.getUser();
-
+    this.updateLogoutTimeout_(this.user);
     this.responseType = 'token';
     this.redirectUri = `${location.origin}${location.pathname}`;
     this.scope = 'openid+profile';
   }
 
+  /**
+   * Set or clear auto logout timer.
+   * @param claims
+   */
+  updateLogoutTimeout_(claims) {
+    if (this.expireTimer_) {
```

Comment:
will it work ? shouldn't `this.expireTimer_` be set to `null` after `clearTimeout(this.expireTimer_);` ?